shared/debrid.py

```diff
@@ -177,7 +177,7 @@ def _getInstantAvailability(self, refresh=False):
             torrentHash = self.getHash()
             self.print('hash:', torrentHash)
 
-            if len(torrentHash) != 40:
+            if len(torrentHash) != 40 or True:

```

Comment:
this will always pass.. so `len(torrentHash)` won't ever matter

Reply:
Get out! We don't do good code in this house!